src/main/java/com/zaxxer/hikari/pool/ProxyConnection.java

```diff
@@ -455,7 +455,7 @@ private static Connection getClosedConnection()
             public Object invoke(Object proxy, Method method, Object[] args) throws Throwable
             {
                final String methodName = method.getName();
-               if ("abort".equals(methodName)) {
+               if ("close".equals(methodName) || "abort".equals(methodName)) {
```

Comment:
What's going on here?  Multiple-close is already handled -- see [``testDoubleClose()``](https://github.com/brettwooldridge/HikariCP/blob/dev/src/test/java/com/zaxxer/hikari/pool/TestConnections.java#L220).  This proxy only handles methods not intercepted by ``ProxyConnection``.  The ``close()`` method is already intercepted and is pointless here.